CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
```

Comment:
We should do this in two commits. One without the "unreleased" section and one with. Then we will tag the first commit with v0.6.0.